discovery_child_development/utils/google_utils.py

```diff
@@ -79,3 +81,20 @@ def create_client() -> bigquery.Client:
     # Initialize a client with the provided credentials
     client = bigquery.Client(credentials=credentials)
     return client
+
+
+def access_google_sheet(google_credentials_json, sheet_id, sheet_name):
```

Comment:
I think you could make this even more convenient if you load in the credentials at the top of this module - and then provide them as the default variable value. In that way you won't need to write the code for loading credentials every time you want to access google sheets (as you now do, eg in the taxonomy util)

Also, types and a docstring would be great!